modules/web/src/app/shared/components/application-list/component.ts

```diff
@@ -312,6 +312,7 @@ export class ApplicationListComponent implements OnInit, OnDestroy {
         this.applicationsMethodMap = {};
         this.applicationsSourceMap = {};
         this.applicationsStatusMap = {};
+        // eslint-disable-next-line complexity
```

Comment:
I can't really restructure this function due to code freeze so disabling linting here for now.